packages/arb-token-bridge-ui/src/components/common/Header.tsx

```diff
@@ -211,16 +190,30 @@ export function HeaderContent({ children }: { children: React.ReactNode }) {
 }
 
 export function Header() {
```

Comment:
I don't like this change. `Header` was designed so it doesn't have to do anything with wallet connection, and different content can be injected through portals, because at some point we want to reuse it across different apps.